.github/workflows/build-lint-test.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           cache-dependency-path: tests/node/yarn.lock
       - name: Install
         working-directory: tests/node
-        run: yarn install --immutable --immutable-cache
+        run: yarn install --immutable
```

Comment:
do we need the `--immutable` ? The CI uses it by default?
> Yarn detected that the current workflow is executed from a public pull request. For safety the hardened mode has been enabled.
It will prevent malicious lockfile manipulations, in exchange for a slower install time. You can opt-out if necessary; check our documentation for more

Reply:
I think that's ok to have it being explicit yes!